### PR TITLE
Tests: find qemu on (open)SUSE systems

### PR DESCRIPTION
**Pull request by danimo**

### CODE REVIEW

**Comment by haraldh** [comment]
@centos-ci
OK to test

**Comment by danimo** [comment]
@centos-ci
OK to test

**Comment by danimo** [comment]
retest this please

**Comment by haraldh** [comment]
@centos-ci 
retest this please

File: test/run-qemu [hunk 1]
```diff
@@ -8,6 +8,7 @@ $(lsmod | grep -q '^kqemu ') && BIN=/usr/bin/qemu && ARGS="-kernel-kqemu "
 [[ -c /dev/kvm && -x /usr/bin/kvm ]] && BIN=/usr/bin/kvm && ARGS=""
 [[ -c /dev/kvm && -x /usr/bin/qemu-kvm ]] && BIN=/usr/bin/qemu-kvm && ARGS=""
 [[ -c /dev/kvm && -x /usr/libexec/qemu-kvm ]] && BIN=/usr/libexec/qemu-kvm && ARGS=""
+[[ -c /dev/kvm && -x /usr/bin/qemu-system-$(uname -i) ]] && BIN=/usr/bin/qemu-system-$(uname -i) && ARGS=""
```
Comment (haraldh) [inline test/run-qemu]: should ARGS be "-enable-kvm" ? as you check for /dev/kvm

Comment (danimo) [inline test/run-qemu]: You are right, I submitted an update

**Comment by danimo** [comment]
@centos-ci
retest this please

**Pull request merged**